docs/Documentation.md

```diff
@@ -90,10 +90,57 @@ mongo_db_bundle:
       client_name: ~
       database_name: ~
 
+  # Service reference to provide URI options - see example below
+  uriOptions: 'App\Services\UriOptionsProvider' # default null
+  
   # Service reference to provide driver options - see example below
   driverOptions: 'App\Services\DriverOptionsProvider' # default null
 ```
 
+### Uri options
+
+You might need to specify some URI options for constructing the `MongoDB\Client`. Read the [reference] for a complete
+explanation of all the available options.
+
+Implement `UriOptionsInterface` and declare the class as a Symfony service.
+
+```php
+namespace App\Services;
+
+use Facile\MongoDbBundle\Services\UriOptions\UriOptionsInterface;
+
+final class MyCustomUriOptionsProvider implements DriverOptionsInterface
+{
+    /** @var string */
+    private $appname;
+    
+    public function __construct(string $appname) {
+        $this->appname = $appname;
+    }
+
+    public function buildDriverOptions(array $clientConfiguration) : array {
+        $clientConfiguration['appname'] =  $this->appname;
+        return $clientConfiguration;
+    }
+}
+```
+
+```yaml
+# config/services.yaml
+App\Services\MyCustomUriOptionsProvider:
+  arguments:
+    $appname: 'APPNAME'
```

Comment:
I think we could drop this argument from the sample, as it's not a minimum working sample.
Just hard-code the app name in the service implementation. We assume the users know exactly which are the best practices to inject configuration in Symfony services.

Reply:
Are you proposing to put more exhaustive arguments in the sample?

Reply:
nope, I think it's ok to just hardcode `APPNAME` here https://github.com/facile-it/mongodb-bundle/pull/126/files/d15bc89b5924c8300810d9bbedc97d8125e4f10b#diff-7323a8bcdc55fd8130bd5f6b80ea950617519487ecd175d7ab60bfe8b07396bfR121